app/models/financialDetails/CreditType.scala

```diff
@@ -53,6 +53,7 @@ object CreditType {
   private val mfaCredit = Range.inclusive(4004, 4025)
     .filterNot(_ == 4010).filterNot(_ == 4020).map(_.toString)
     .toList
+  // TODO: This should be "0060", not "4920" or "4930"
```

Comment:
do we need a tech debt story for this change or this is still some work in progress ?

Reply:
It needs a bug ticket

Reply:
Please feel free to raise a bug ticket so we can start work on this in the next sprint